website/README.md

```diff
@@ -9,4 +9,4 @@ The actual contents are generated from the markdown files on the browser side:
 
 The site is previewable and deployable with just the single [`index.html`](./index.html).
 
-No dependency on any templating engine currently, but eventually we may adopt docsy or something else similar.
+No dependency on any templating engine currently, but eventually we may adopt docsy or something else similar.
```

Comment:
I meant to keep L1-L3.
Other lines have to be removed

Reply:
My bad. Done the changes